internal/schemautil/user_config.go

```diff
@@ -20,7 +22,7 @@ func GenerateServiceUserConfigurationSchema(t string) *schema.Schema {
 		Type:             schema.TypeList,
 		MaxItems:         1,
 		Optional:         true,
-		Description:      fmt.Sprintf("%s user configurable settings", strings.Title(t)),
+		Description:      fmt.Sprintf("%s user configurable settings", cases.Title(language.Und, cases.NoLower).String(t)),
```

Comment:
it will make all words in the description field start with the capital, we will have something like `Allow incoming connections from ...` -> `Allow Incoming Connections From ....` 

Lets not alter the description field in this way.

Reply:
Checked the code from above, we only pass a a service name here as `t`, so we are good to go!

Reply:
`strings.Title` does that already anyway